Gopkg.toml

```diff
@@ -65,15 +65,19 @@ ignored = [
 
 [[constraint]]
   name = "github.com/gravitational/satellite"
-  version = "=5.5.4"
+  version = "=6.1.1"
+
+[[constraint]]
+  name = "github.com/miekg/dns"
```

Comment:
That's an unfortunate consequence of satellite not pinning the version of `miekg/dns` - will need to fix that in another update.